debian/sourcemapper_test.go

```diff
@@ -0,0 +1,30 @@
+package debian
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/quay/claircore/test/integration"
+	"github.com/quay/zlog"
+)
+
+func TestCreateSourcesMap(t *testing.T) {
+	integration.Skip(t)
+	ctx := zlog.Test(context.Background(), t)
+	mapper := NewSourcesMap(Bullseye, http.DefaultClient)
+
+	err := mapper.Update(ctx)
+	if err != nil {
+		t.Fatalf("unexpected error %v", err)
+	}
+	opensshBinaries := mapper.Get("openssh")
+	if len(opensshBinaries) != 8 {
+		t.Fatalf("expected 8 binaries related to openssh found %d found %v", len(opensshBinaries), opensshBinaries)
+	}
+
+	tarBinaries := mapper.Get("tar")
+	if len(tarBinaries) != 2 {
+		t.Fatalf("expected 2 binaries related to openssh found %d found %v", len(tarBinaries), tarBinaries)
+	}
+}
```

Comment:
Let me know what you think of the unit-test. Seemed eventually to me simpler to manipulate the client rather than create a test-server due to no having access to the generated URLs.